tools/ci_build/github/linux/docker/manylinux/install_deps.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
```

Comment:
how do install_deps.sh and install_centos.sh differ from the ones in the docker/scripts directory.
we have a lot of scripts and it's difficult to know what is used for what, and there seems to be some duplication as well.

Reply:
This install_deps.sh  needs to build nodejs from source, because the official binary doesn't support CentOS 6.  And it installs python packages for every python version we need. 

I'll remove the other " install_centos.sh" when I finish the migration. We don't need two.